lib/ansible/modules/user.py

```diff
@@ -855,7 +855,8 @@ def modify_user_usermod(self):
 
         if self.expires is not None:
 
-            current_expires = int(self.user_password()[1])
+            current_expires = self.user_password()[1] or '0'
```

Comment:
This needs to be set to `0` so that the rest of the logic continues to work as expected. Setting this to `-1` results in changes being made on every task in certain scenarios.